### ISSUE TITLE
[proposal] Change repository name

### ISSUE THREAD

**Issue by freesky-edward** [huaweicloud/external-sfs#11]
This is a proposal than an issue, the name of this repository is fine as a project under kubernetes-incubator, but not clear here. to make the name clear, I suggest to change the repository name to following candidate:
1. k8s-external-huaweicloud-sfs
2. huaweicloud-k8s-external-sfs
3. huaweicloud-k8s-external-sfs-provisioner
4. ....

please feel free to reply your thoughts or any suggestion.

**Comment by edisonxiang** [comment]
Thanks your suggestion. Actually I am hesitating what name is suitable for this repository.
In the original idea, this repository is aim to openshift and k8s so it is better to remove k8s in the name. secondly, i am not sure whether we will add other sfs features in this repository, such as flex or csi plugin so it is kept as not include the provisioner. Maybe huaweicloud-external-sfs is the suitable name for this repository, or this repository is already in huaweicloud community so we can leave as it is?

**Comment by freesky-edward** [comment]
do you mean this repository can only run with openshift instead of  common k8s?

**Comment by edisonxiang** [comment]
nope, there are some differences between openshift and kubernetes, especially in [rights management.](https://github.com/huaweicloud/external-sfs/blob/master/README.md).
